policy-controller/core/src/http_route.rs

```diff
@@ -9,7 +9,7 @@ pub use http::{
 use regex::Regex;
 use std::num::NonZeroU16;
 
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
```

Comment:
What would it mean to create a default `HttpRoute`? I assume this is to support `..Default::default()`? IMO, the syntactic sugar isn't worth it if the empty value type isn't coherent.